replicate/prediction.py

```diff
@@ -603,6 +610,18 @@ async def async_cancel(self, id: str) -> Prediction:
         return _json_to_prediction(self._client, resp.json())
 
 
+def _create_prediction_headers(
+    *,
+    block: Optional[bool] = None,
```

Comment:
Why `block` and not `wait` or `sync`?

Reply:
Because bike sheds.

Reply:
`block` sounds like a "bad" thing to me out of context. What am I blocking, etc?

Reply:
Good news! With the move from `X-Sync: true` to `Prefer: wait`, this can now called `wait`.